mars/core.py

```diff
@@ -322,7 +322,7 @@ def params(self):
         p = {
```

Comment:
How about just return the tail node's `params`?

Reply:
Done